src/main/java/io/confluent/idesidecar/restapi/auth/AuthErrors.java

```diff
@@ -79,7 +79,6 @@ public AuthErrors withoutTokenRefresh() {
   }
 
   public record AuthError(
-      @JsonProperty(value = "created_at") Instant createdAt,
```

Comment:
The meat of the PR. Remove the createdAt field causing connections with "auth errors" for their kafka or schema registry to jitter over websocket every 5s check pass.